### PR TITLE
Feature/e2e fr version

### PR DESCRIPTION
**Pull request by Iloveall**
## Description

Fixes [1715](https://github.com/France-ioi/AlgoreaFrontend/issues/1715)

## Notes (out of scope, known isues, hints for reviewing code, ...)  (optional)

...

### CODE REVIEW

File: e2e/translates.spec.ts
Comment (smadbe) [inline e2e/translates.spec.ts:1]: This test is not a test of translation but a specific check for the "fr" translation, so I think the file name should be more explicit. For instance, `fr-translation.spec.ts`.

File: e2e/translates.spec.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import { test, expect } from './groups/fixture';
+import { initAsUsualUser } from 'e2e/helpers/e2e_auth';
+import { apiUrl } from 'e2e/helpers/e2e_http';
+
+test.use({ baseURL: 'http://localhost:4100' });
```
Comment (smadbe) [inline e2e/translates.spec.ts:5]: You should specify that the "fr" translation has been run on this port.

**Review by smadbe — APPROVED** [review]

File: e2e/translates.spec.ts (hunk 1 above)
Comment (Iloveall) [reply e2e/translates.spec.ts]: Fixed

**Comment by smadbe** [comment]
[auto-generated message] E2E test report on: https://algorea-static.s3.eu-central-1.amazonaws.com/branch/feature/e2e-fr-version/playwright-report/index.html

**Comment by smadbe** [comment]
[auto-generated message] E2E test report on: https://algorea-static.s3.eu-central-1.amazonaws.com/branch/feature/e2e-fr-version/playwright-report/index.html

**Pull request merged**